.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-PLEASE READ before removing!!
+# PLEASE READ before removing
```

Comment:
Here we are loosing `!!` due to [md041](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md041---first-line-in-file-should-be-a-top-level-header)